github/github.go

```diff
@@ -481,7 +489,19 @@ func (r *Response) populatePageValues() {
 			if err != nil {
 				continue
 			}
-			page := url.Query().Get("page")
+
+			q := url.Query()
+
+			if cursor := q.Get("cursor"); cursor != "" {
+				for _, segment := range segments[1:] {
+					switch strings.TrimSpace(segment) {
+					case `rel="next"`:
+						r.Cursor = cursor
+					}
+				}
+			}
```

Comment:
Are `"cursor"` and `"page"` mutually exclusive?  (I think the answer is "yes" but I could be wrong.)

If so, do you want to add a `continue` within this `if` block just to make that clear?
I'm actually fine either way... your call.

Reply:
Thanks! Yes, I believe so, although it isn't documented anywhere. Added `continue` in 72c3380